protos/bottle/catalog/v1/addon.proto

```diff
@@ -6,6 +6,8 @@ message Addon {
   enum AddonCategory {
     ADDON_CATEGORY_UNSPECIFIED = 0;
     ADDON_CATEGORY_RUSH_ASSEMBLY = 1;
+    ADDON_CATEGORY_HANDLING = 2;
```

Comment:
`Shipping label (admin fee)  OH010000	Handling only charges (separately identified from shipping)`

I figure we could just put this as a handling fee because that is what it is.  Gives us the option to use it down the road if we need to.